Makefile

```diff
@@ -1,67 +1,38 @@
-PYTHON_VERSION = python3.8
-VIRTUALENV := .venv
-
-.PHONY: virtualenv
```

Comment:
I'd quite like to keep these recipes if we can because they do make life a lot easier. I think we definitely don't want to be managing `requirements_dev.txt` manually

Reply:
I can restore it for sure.